### PR TITLE
[TD-13118]<feature>:nodejs stmt supports bind_param()

### PR DESCRIPTION
**Pull request by xleili**

### CODE REVIEW

File: src/connector/nodejs/examples/taosBindParamSample.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+                    `i32 int,`+
+                    `i64 bigint,`+
+                    `f32 float,`+
+                    `d64 double,`+
+                    `bnr binary(20),`+
+                    `blob nchar(20),`+
+                    `u8 tinyint unsigned,`+
+                    `u16 smallint unsigned,`+
+                    `u32 int unsigned,`+
+                    `u64 bigint unsigned);`;
```
Comment (zitsen) [inline src/connector/nodejs/examples/taosBindParamSample.js:41]: json type should be supported in stmt bind too.

Comment (xleili) [reply src/connector/nodejs/examples/taosBindParamSample.js:41]: I will support bind json type in next pr.

**Review by zitsen — APPROVED** [review]
LGTM

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**